### PR TITLE
LPS-185518 Create a SF check for replace with searchContainer.setResultsAndTotal

### PR DESCRIPTION
**Pull request by TamyrisTorres**
LPS-185518 https://issues.liferay.com/browse/LPS-185518

### CODE REVIEW

File: modules/util/source-formatter/src/test/java/com/liferay/source/formatter/processor/UpgradeSourceProcessorTest.java [hunk 1]
```diff
@@ -64,13 +64,15 @@ public void testUpgradeJavaCheck() throws Exception {
 	}
 
 	@Test
-	public void testUpgradeJavaExtractTextMethodCheck() throws Exception {
-		test("upgrade/UpgradeJavaExtractTextMethodCheck.testjava");
+	public void testUpgradeJavaServiceTrackerListCheck() throws Exception {
+		test("upgrade/UpgradeJavaServiceTrackerListCheck.testjava");
```
Comment (natocesarrego) [inline modules/util/source-formatter/src/test/java/com/liferay/source/formatter/processor/UpgradeSourceProcessorTest.java:68]: @TamyrisTorres, is this change really related to this pull?

In addition, this part of the commit should actually be part of the second commit of this pull, right?

**Comment by TamyrisTorres** [comment]
@natocesarrego, Can you review it again, please?

**Comment by TamyrisTorres** [comment]
ci:test:sf

**Comment by natocesarrego** [comment]
@TamyrisTorres, good job!

Please forward to Kevin's review.

Thanks
